app/authenticated/projects/new/route.js

```diff
@@ -0,0 +1,25 @@
+import { hash } from 'rsvp';
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+    authzStore: service('authz-store'),
+    model: function () {
+        const project = this.get('authzStore').createRecord({
+            type: `project`,
+            name: '',
+        });
+        const projectRoleTemplateBindings = [{
```

Comment:
Default the list to empty here (and call addMember after render in the component if you want an empty row).  Doing it here creates a duplicate copy of what an empty binding looks like.